numpy/lib/shape_base.py

```diff
@@ -850,7 +850,12 @@ def tile(A, reps):
     except TypeError:
         tup = (reps,)
     d = len(tup)
-    c = _nx.array(A, copy=False, subok=True, ndmin=d)
+    if all((x == 1 for x in tup)) and isinstance(A, _nx.ndarray):
+        # Fixes the problem that the function does not make a copy if A is a
+        # numpy array and the repetitions are 1 in all dimensions
+        c = _nx.array(A, copy=True, subok=True, ndmin=d)
```

Comment:
Rather than processing a copy you can simply return it, as none of the code below is going to do anything to it.